vendor/assets/stylesheets/materialize/_variables.scss

```diff
@@ -110,6 +110,10 @@ $chip-bg-color: #e4e4e4 !default;
 $chip-border-color: #9e9e9e !default;
 $chip-selected-color: #26a69a !default;
 $chip-margin: 5px !default;
+$chip-height: 32px !default;
+$chip-line-height: 32px !default;
+$chip-font-size: 13px !default;
+$chip-font-weight: 500 !default;
```

Comment:
This is the default library, no changes should be done here. Please move it there, if needed: app/assets/stylesheets/mtl/extend/_chips.scss